### ISSUE TITLE
Mental Disorder and WCAG 3.0

### ISSUE THREAD

**Issue by Rumperuu** [w3c/wcag3#15]
I have recently been exploring the issue of mental disorders and accessibility. This is an area that is, as far as I can tell, currently very underserved.

I reviewed the proposed WCAG 3 Working Draft in the process of doing this and have a few comments:

- The inclusion of the Mental Health functional category is a great step, though it currently only contains a single functional need targeting anxiety and obsessive-compulsive disorders:
    - The ICD-11 lists 19 basic categories of 'mental, behavioural or neurodevelopmental disorders'. Some of these are likely beyond the scope of WCAG, and some would be (wholly or partly) covered by the other functional needs, but some areas (like addiction) are potentially missed in the current list as drafted.
    - For a good catch-most functional need under the Mental Health category, I would suggest the addition of something like 'use without exacerbating pre-existing disorders'
    - I was wondering whether this functional category has been discussed much yet, or was it added as a placeholder with the intention of fleshing out the needs later? I see that the Mental Health Subgroup [have a current to-do](https://www.w3.org/WAI/GL/task-forces/coga/wiki/Subgroups/Mental_Health#To_Do.27s) to look into 'mental health from a functional needs perspective' which I assume would feed into this.
- I believe the functional need to 'use without harm or risk' is ambiguous and potentially problematic whichever way it is interpreted:
    - If interpreted as 'use without harm or risk to the user', it can interact with the need to 'use with limited judgement' in a way that I think could pose a risk of unduly restricting user agency (i.e., limiting accessibility in the name of accessibility) and possible infantilisation.
    - If interpreted as 'use without harm or risk to others', I think it would be stepping well beyond the scope of accessibility and into thorny issues of content moderation and censorship.
- The intended reading of the 'use without harm or risk' functional need is not clarified by the draft guidelines, which contain the equally-ambiguous 'prevent harm'.
    - However, t[here](https://bengoldsworthy.net/blog/posts/mental-disorders-and-accessibility/) is a separate 'harm from motion' guideline that is much more clearly defined, and I think this shows a good level of specificity to break the 'prevent harm' down to
- I believe that an additional functional category of 'Technical' is necessary, which would include functional needs such as 'use with limited Internet bandwidth' and 'use with an unreliable Internet connection' and would serve (amongst other things) as a proxy for ensuring financial accessibility.

Those are my key takeaways; if anyone wants to read my full article it's available [here](https://bengoldsworthy.net/blog/posts/mental-disorders-and-accessibility/).

**Comment by colabottles** [comment]
> I have recently been exploring the issue of mental disorders and accessibility. This is an area that is, as far as I can tell, currently very underserved.

Absolutely. That said, the work that COGA has done/is doing may address some of your comments. The FAST subgroup will be meeting and [the current FAST Editor's Draft - ED](https://w3c.github.io/fast/) is still a long way to being completed and a lot of work and research has been put into the document. 

> The inclusion of the Mental Health functional category is a great step, though it currently only contains a single functional need targeting anxiety and obsessive-compulsive disorders: 

We have yet to go through and pick apart the generalized category of Mental Health. This is noted and many more user and functional needs can, should, and will be brought to the forefront, I am sure. We have this as a placeholder until the FAST subgroup meets to discuss where we go from the ED.

> I believe the functional need to 'use without harm or risk' is ambiguous and potentially problematic whichever way it is interpreted: 

Again, if you look at the FAST ED, this is a placeholder until we can meet to discuss specifics and break things down to a more granular level. We do have a couple of user needs in the placeholders and I am sure there will be more to come.

> I believe that an additional functional category of 'Technical' is necessary...

Personally, I would place this in "Operable" as a subcategory. 'Technical' is too broad of a term for my liking. Others may have differing opinions. "Harm from Motion" was something I spoke about during the Functional Needs meetings because of my experience with motion and migraine headaches being caused by motion and animation on the Web 

There is a lot of work that needs to be done and will be starting up soon. We are still in the beginning stages of addressing this through FAST. I am sure this discussion will be seen and shared and discussed as we move forward to bring the Mental Health piece to FAST and WCAG 3.

_Note: These are my views and opinions and does not necessarily reflect the views of other members of the W3C or the W3C as an organization as a whole._

**Comment by rachaelbradley** [comment]
I think this is OK as an issue for the time being.  The guidelines as currently written are placeholder text. They give a general idea of what is coming but don't convey the details. The initial conversation on this area will be happening through subgroups and leverage the work from the COGA task force's mental health subgroup.  I will bring this issue to those group's attention.

**Comment by lseeman** [comment]
Hi Ben
The coga task force ( - see https://www.w3.org/WAI/GL/task-forces/coga/wiki/Main_Page)
 is drafting suggestions based on our literary review for the next version of making content useable for people with learning and cognitive disabilities - see https://www.w3.org/TR/coga-usable/. We have increased our scope to included mental health as there is such a large overlap were mental health affects cognitive function. 
We have also made suggestions for changes to FAST and will be integrating our work as much as we can into wcag 3.
@rashmi will read over ypu blog post to see if we have missed anything.

**Comment by ajstamm** [comment]
As someone with low bandwidth and accustomed to living in rural areas, I'd love to see all websites (and internet software like Teams or Zoom) implement this:
> I believe that an additional functional category of (as an example) 'Technical' is necessary, which would include functional needs such as 'use with limited Internet bandwidth' and 'use with an unreliable Internet connection' and would serve (amongst other things) as a proxy for ensuring financial accessibility.

**Comment by cwadamsoforacle** [comment]
I will follow up with COGA to see if they have produced the issue papers on this topic.
